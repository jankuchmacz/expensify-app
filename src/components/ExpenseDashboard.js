import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpenseSummary from './ExpenseSummary';

const ExpenseDashoboardPage = () => (
    <div>
        <ExpenseSummary/>
        <ExpenseListFilters />      
        <ExpenseList />
    </div>
);
export {ExpenseDashoboardPage as default}