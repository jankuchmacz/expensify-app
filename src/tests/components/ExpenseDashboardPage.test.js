import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboard from "../../components/ExpenseDashboard";

test('Expense Dashboard renders correctly', () => {
    const wrapper = shallow(<ExpenseDashboard/>);
    expect(wrapper).toMatchSnapshot();
})