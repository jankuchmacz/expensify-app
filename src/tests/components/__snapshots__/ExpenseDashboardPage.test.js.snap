// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expense Dashboard renders correctly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
